templates/rustdoc/topbar.html

```diff
@@ -130,17 +125,7 @@
                             <li class="pure-menu-item">
                                 <div class="pure-menu pure-menu-scrollable sub-menu" tabindex="-1">
                                     <ul class="pure-menu-list">
-                                        {%- for dep in krate.dependencies -%}
-                                            <li class="pure-menu-item">
-                                                <a href="/{{ dep[0] }}/{{ dep[1] }}" class="pure-menu-link">
-                                                    {{ dep[0] }} {{ dep[1] }}
-                                                    <i class="dependencies {{ dep[2] | default(value='') }}">{{ dep[2] | default(value="") }}</i>
-                                                    {%- if dep[3] -%}
-                                                    <i> optional </i>
-                                                    {%- endif -%}
-                                                </a>
-                                            </li>
-                                        {%- endfor -%}
+                                        {% call macros::dependencies_list(link_prefix="", dependencies=krate.dependencies, if_empty="") %}
```

Comment:
It seems like this shows all dependencies mistakenly as optional?

<img width="364" alt="grafik" src="https://github.com/rust-lang/docs.rs/assets/540890/e2f0a7e0-f906-4232-992e-fba393b13081">

to compare:
https://docs.rs/regex/1.10.5/regex/